### DISCUSSION TITLE
Ibax ı nasıl bilirdik .d

### DISCUSSION THREAD

**Discussion by ghost** [IBAX-io/go-ibax#1211] [Soru Cevap - Türkçe]
Youtuber lerden gördüm geldim Sağlam projeymiş .d gerçekten sağlammı?

**Comment by Skips1** [comment]
Kanka Bir çok ki ordan gördü zaten Trnin niye bu kadar kalabalık olduğunu anlamıssındır. Bu proje eğer güçlenirse Tr dekiler iyi kazanacaktır bekleyip göreceğiz herşey için çok erken kardeşim benim.

**Reply by Skips1** [reply]
Answer lediğin için şağol kanka .d

**Reply by ghost** [reply]
Ne demek 🦖
